### DISCUSSION TITLE
Chart Widget style error

### DISCUSSION THREAD

**Discussion by MuhamadMwaas** [filamentphp/filament#14325] [Help] [answered]
### Package

Widgets

### Package Version

v3.2.0

### How can we help you?

![لقطة الشاشة 2024-09-23 155252](https://github.com/user-attachments/assets/edcbff52-e119-4012-802e-8ac0579f04a2)
I have an issue with the charts in Filament PHP. When I use the chart, it appears like this, as shown in the picture. There is an error in its formatting, and I have tried everything I can but haven't succeeded. Note that the issue only appears on the server, while it works without problems locally.

**Comment by leandrocfe** [comment, marked as answer]
https://github.com/filamentphp/filament/releases/tag/v3.2.114
_Lock Livewire to max v3.5.6 due to bug introduced #14315._

**Comment by leandrocfe** [comment]
Did you run `composer update`?

**Reply by MuhamadMwaas** [reply]
fixed by Lock Livewire to max v3.5.6
